build/jobs/run-tests.yml

```diff
@@ -0,0 +1,132 @@
+parameters:
+- name: version
+  type: string
+- name: keyVaultName
+  type: string
+- name: runIntegration
+  type: boolean
+  default: true
+- name: runE2E
+  type: boolean
+  default: true
+
+jobs:
+- job: "integrationTests"
+  condition: eq('${{ parameters.runIntegration }}', true)
+  pool:
+    vmImage: 'vs2017-win2016'
+  steps:
+  - task: DownloadBuildArtifacts@0
+    inputs:
+      buildType: 'current'
+      downloadType: 'single'
+      downloadPath: '$(System.ArtifactsDirectory)'
+      artifactName: 'IntegrationTests'
+
+  - task: UseDotNet@2
+    inputs:
+      version: '3.1.100' 
```

Comment:
Should be like this one https://github.com/microsoft/fhir-server/blob/master/build/build.yml#L9
`useGlobalJson: true` I think there should be an input for where to find it as an artifact

Reply:
Changed